### PR TITLE
BLOCKS-272 Create more Catblocks tests

### PR DESCRIPTION
**Pull request by MozartPanda**
Added new parser test cases.
https://jira.catrob.at/browse/BLOCKS-272

### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by liflyanna — APPROVED** [review]
LGTM

File: test/jsunit/parser/parser.test.js [hunk 1]
```diff
@@ -2818,5 +2818,611 @@ describe('Catroid to Catblocks parser tests', () => {
...
+            })
+          ])
+        })
+      );
+
+      expect(mapKeys).toEqual([categoryName]);
+      expect(mapValues).toEqual([`join( ${first} , ${second} , ${third} )`]);
+    });
+
+    test('multiple formular', async () => {
```
Comment (bprattes) [inline test/jsunit/parser/parser.test.js:2922]: please fix typo: formula instead of formular

File: test/jsunit/parser/parser.test.js [hunk 2]
```diff
@@ -2818,5 +2818,611 @@ describe('Catroid to Catblocks parser tests', () => {
       expect(mapKeys).toEqual([categoryName]);
       expect(mapValues).toEqual([expectedOutput]);
     });
+
+    test('formular with 3 arguments', async () => {
```
Comment (bprattes) [inline test/jsunit/parser/parser.test.js:2822]: please fix typo: formula instead of formular

File: test/jsunit/parser/parser.test.js [hunk 3]
```diff
@@ -2818,5 +2818,611 @@ describe('Catroid to Catblocks parser tests', () => {
...
+        ` ${first[7].toLowerCase().replaceAll('_', ' ')} `,
+        ` ${first[8].toLowerCase().replaceAll('_', ' ')} `,
+        ` number of current touches `,
+        `index of current touch( ${first[10]} )`,
+        ` ${first[11].toLowerCase()} `
+      ];
+      const xmlString = `<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+      <program>
+        <header>
+          <programName>formulars</programName>
```
Comment (bprattes) [inline test/jsunit/parser/parser.test.js:2971]: please fix typo: formula instead of formular

**Review by bprattes — CHANGES_REQUESTED** [review]

**Review by bprattes — APPROVED** [review]
LGTM

**Pull request merged**